### ISSUE TITLE
The encoding of the default admin interface method has changed.

### ISSUE THREAD

**Issue by TrustHenry** [bosagora/agora#2286]
By default, the return values of the interface methods are serialized as JSON text and sent back to the REST client.

Affected by the application of the serialization policy.
`@serializationPolicy!(Base64ArrayPolicy)`

Affected Methods
These methods should be serialized as JSON text.

```
public string loginQR (@viaHeader("Content-Type") out string contentType,
                           @viaHeader("Vary") out string vary);

public string encryptionKeyQR (string app, ulong height,
                                   @viaHeader("Content-Type") out string contentType,
                                   @viaHeader("Vary") out string vary);
```

**Comment by ferencdg** [comment]
I think your issue is that there are escaped double quotes (\\") in the output you receive, but that has nothing to do with Base64 serialization. 

Please let me know 
1. what the output you receive
2. what the output you **want** to receive

from the loginQR function

**Comment by ferencdg** [comment]
@TrustHenry after https://github.com/bosagora/agora/pull/2300 is merged, you will need to send your requests from the client with 

```
Accept: image/svg+xml
```
set

**Comment by Geod24** [comment]
Fixed by #2300
